pages/linux/telinit.md

```diff
@@ -0,0 +1,28 @@
+# telinit
+
+> Change SysV runlevel.
+> Since the concept SysV runlevels is obsolete the runlevel requests will be transparently translated into systemd unit activation requests. More information: <https://manned.org/telinit>.
```

Comment:
```suggestion
> Since the concept of SysV runlevels is obsolete the runlevel requests will be transparently translated into systemd unit activation requests. 
> More information: <https://manned.org/telinit>.
```